coworking/demo/website_view.xml

```diff
@@ -0,0 +1,746 @@
+              <div class="container s_allow_columns">
+                <div class="s_social_media text-center mb-4 o_not_editable" data-snippet="s_social_media" data-name="Social Media">
+                  <h5 class="s_social_media_title d-none">Follow us</h5>
+                  <a href="/website/social/facebook" class="s_social_media_facebook" target="_blank" aria-label="Facebook">
+                    <i class="fa fa-facebook rounded-circle rounded shadow-sm o_editable_media"/>
+                  </a>
+                  <a href="/website/social/twitter" class="s_social_media_twitter" target="_blank" aria-label="X">
+                    <i class="fa fa-twitter rounded-circle rounded shadow-sm o_editable_media"/>
+                  </a>
+                  <a href="/website/social/linkedin" class="s_social_media_linkedin" target="_blank" aria-label="LinkedIn">
+                    <i class="fa fa-linkedin rounded-circle rounded shadow-sm o_editable_media"/>
+                  </a>
+                </div>
+                <p class="text-center mb-1">250 Executive Park Blvd, Suite 3400 • San Francisco CA 94134 • United States</p>
+                <ul class="list-inline text-center">
+                  <li class="list-inline-item mx-3">
+                    <i class="fa fa-1x fa-fw fa-phone me-2"/>
+                    <span class="o_force_ltr">
+                      <a href="tel:+1 555-555-5556">+1 555-555-5556</a>
+                    </span>
+                  </li>
+                  <li class="list-inline-item mx-3">
+                    <i class="fa fa-1x fa-fw fa-envelope me-2"/>
+                    <span>
+                      <a href="mailto:info@yourcompany.example.com">info@yourcompany.example.com</a>
+                    </span>
+                  </li>
+                </ul>
+                <div class="row">
+                  <div class="col-lg-2 offset-lg-5 o_colored_level">
+                    <a href="/" class="o_footer_logo logo">
+                      <img src="/web/image/coworking.ir_attachment_1" alt="" data-attachment-id="1349" class="img-fluid mx-auto img o_we_custom_image" data-format-mimetype="image/webp" data-original-id="1349" data-original-src="/web/image/1349-e7725da7/coworking.png" data-mimetype-before-conversion="image/png" data-resize-width="512" data-mimetype="image/webp" loading="lazy"/>
+                    </a>
+                  </div>
+                </div>
+              </div>
+            </section>
+          </div>
+        </xpath>
+      </data>
+    </field>
+    <field name="inherit_id" ref="website.layout"/>
+    <field name="key">website.template_footer_centered</field>
+    <field name="mode">extension</field>
+    <field name="name">Centered</field>
+    <field name="type">qweb</field>
+    <field name="website_id" ref="website.default_website"/>
+  </record>
+  <record id="contactus" model="ir.ui.view">
```

Comment:
I meant that this one is probably useless, right?
the contactus should target /become-a-member page